include/libdnf5/conf/config_main.hpp

```diff
@@ -44,6 +44,14 @@ class LIBDNF_API ConfigMain : public Config {
 
     OptionNumber<std::int32_t> & get_debuglevel_option();
     const OptionNumber<std::int32_t> & get_debuglevel_option() const;
+    /// @deprecated errorlevel option does nothing and is scheduled for removal
+    __attribute((deprecated("errorlevel option does nothing and is scheduled for removal")))
```

Comment:
It should be `__attribute__((...))`. Sorry for providing a wrong example. See GCC documentation [https://gcc.gnu.org/onlinedocs/gcc/Common-Function-Attributes.html#index-deprecated-function-attribute].

Reply:
I had to use different syntax, I couldn't convince SWIG to accept it.